src/style.css

```diff
@@ -1 +1,34 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 22px;
+  display: inline-block;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16;
+  height: 16;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(/src/images/star-active.svg);
+}
+
+.stars:not(.stars--5) {
+  margin-bottom: 4px;
```

Comment:
Do not ignore comments:
![image](https://github.com/mate-academy/layout_stars/assets/86923089/9999c083-2d7e-4e74-a195-40bcb31f90e3)

Write some answer or ask for more info (in the chat) if something isn't clear :slightly_smiling_face: 
The reason you added it is clear - but because of it tests will fail for example (BTW Backstop Report link doesn't work)

Reply:
We discussed this issue with Nazar during the Q&A session and agreed that there is no need to do it, as the Figma layout has been updated